python_modules/dagster/dagster/seven/__init__.py

```diff
@@ -54,3 +54,20 @@ def reload_module(module):
         return reload_(module)
 
     return reload(module)  # pylint: disable=undefined-variable
+
+
+def is_ascii(str_):
```

Comment:
wowzers

is there some stack overflow somewhere that describes this? Or is it your own divination?

Reply:
i went into trance, and there i found guido, and we discussed it